### PR TITLE
async example

### PR DESCRIPTION
**Pull request by mcollina**
Fixes https://github.com/fastify/point-of-view/issues/82

### CODE REVIEW

**Comment by mcollina** [comment]
cc @red1

File: example-async.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+async function something () {
+  await sleep(1000)
+  return new Date()
+}
+
+fastify.get('/', async (req, reply) => {
+  const t = await something()
+  reply.view('/index.njk', { text: t })
+  return reply
```
Comment (Eomm) [inline example-async.js]: I get this response:
```
{"statusCode":500,"error":"Internal Server Error","message":"Converting circular structure to JSON"}
```

Instead with:
```
return reply.view('/index.njk', { text: t }) // works
```

Comment (mcollina) [reply example-async.js]: you 'll need to update your deps

Comment (Eomm) [reply example-async.js]: You are right 😅

**Review by lilpoozie2005 — APPROVED** [review]

**Pull request merged**